build/release/archive.js

```diff
@@ -0,0 +1,59 @@
+import { readdir, writeFile } from "node:fs/promises";
+import { createReadStream, createWriteStream } from "node:fs";
+import path from "node:path";
+import util from "node:util";
+import os from "node:os";
+import { exec as nodeExec } from "node:child_process";
+import archiver from "archiver";
+
+const exec = util.promisify( nodeExec );
+
+async function md5sum( files, folder ) {
+	if ( os.platform() === "win32" ) {
+		const rmd5 = /[a-f0-9]{32}/;
+		const sum = [];
+
+		for ( let i = 0; i < files.length; i++ ) {
+			const { stdout } = await exec( "certutil -hashfile " + files[ i ] + " MD5", {
+				cwd: folder
+			} );
+			sum.push( rmd5.exec( stdout )[ 0 ] + " " + files[ i ] );
+		}
+
+		return sum.join( "\n" );
+	}
+
+	const { stdout } = await exec( "md5 -r " + files.join( " " ), { cwd: folder } );
```

Comment:
I'm not sure, but let's look into that when we need to support Linux.

Reply:
It will probably be needed when we automate the release via GitHub Actions.

For build breaking Linux would not be acceptable but I guess for a release it is.